core/dbt/include/global_project/macros/materializations/models/table/columns_spec_ddl.sql

```diff
@@ -0,0 +1,13 @@
+{% macro get_columns_spec_ddl() %}
```

Comment:
Reference PR on how to test global macros: https://github.com/dbt-labs/dbt-core/pull/5838/files#diff-649f9b4416cee71ccd51b3b9cfce09b374d393e2a125d7423c9a428928d7bbaf